src/components/ContentUploader/ContentUploader.js

```diff
@@ -276,7 +278,7 @@ class ContentUploader extends Component<Props, State> {
     upload = () => {
         const { items } = this.state;
         items.forEach((uploadItem) => {
-            if (uploadItem.status !== STATUS_IN_PROGRESS) {
+            if (uploadItem.status === STATUS_PENDING) {
```

Comment:
The previous logic would also allow uploads from `STATUS_ERROR` (which might be a valid case?) - `STATUS_COMPLETE` probably wouldn't be a valid starting point.

Reply:
What'll happen when`STATUS_ERROR` is allowed is that the failed uploads will be retryed immediately, before any other pending items being uploaded. Which could potentially cause infinite retry.